k8s/namespaces/ccd/ccd-regr/ccd-regr.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccd-regr
+spec:
+  releaseName: ccd-regr
```

Comment:
Where does the releaseName come from? Proposed by SE? I would prefer naming it like `regtest`, `rt` or `str`. Obviously, it's an environment that we don't want to have regressions but to test if there are any. :)